lib/std/math/big/int.zig

```diff
@@ -670,7 +673,8 @@ pub const Mutable = struct {
                 try t_big.add(r.toConst(), t_big.toConst());
 
                 // u = Cx + Dy, r as u
-                try x.mul(x.toConst(), Cp);
```

Comment:
The issue here is that during the `mul` operation, the current `limbs` array for `x` can be reallocated, which can invalidate the array that is stored in the resulting expression from `x.toConst()`.  My new allocator was setting memory that has been freed to `uninitialized` which exposed this bug.